Formula/libxml2.rb

```diff
@@ -41,7 +41,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_osx
 
-  depends_on :python => :optional
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
I don't know. I'm out-of-the-loop on any internal discussions obviously, but I thought the status quo was that Tim et al. were perfectly happy sticking with the system Python where possible, which is possible here, unless you're running a significantly outdated system?